source/reviews/comments.txt

```diff
@@ -0,0 +1,112 @@
+==========================================
+How To Write Code And Copy Review Comments
+==========================================
+
+Summary
+-------
+
+- Be kind.
+- Explain your reasoning.
+- Balance giving explicit directions with just pointing out problems and letting
+  the author decide.
+- Encourage authors to simplify code or add code comments instead
+  of just explaining the complexity to you.
+- Encourage authors to simplify explanations when appropriate. We're all
+  smart, but smart words take more time to read. Readers prefer
+  conciseness and the simplest explanation in most situations.
+
+Label Your Comments
++++++++++++++++++++
+
+Label comments similar to `Conventional Comments <https://conventionalcomments.org/>`__
+so the author understands this is a suggestion, issue, question, or comment.
+
+- **No "nits"**.
+- Issue: Something is logically, factually, or grammatically incorrect and needs
+  to be corrected. Issues are **always** blocking and no PR should be merged with
+  an open issue.
+- Question: You (the reviewer) have a question about something in a PR.
+  **Do not use** questions as in a Socratic style. If there is a logical, factual, or
+  grammatical error use the proper comment label instead (issue:). Questions must
+  be answered but are **never** blocking, i.e. they do not require a content revision.
+- Comment: You (the reviewer) want to say something about a particular piece
+  of content. Do not use comments as a form of "passive issue". Comments **never** block
+  PR approval and do not require addressing by the author.
+- Suggestion: A way to propose a change as part of a PR comment. It is not required
+  to offer suggestions.
+
+
+.. example::
+
+   .. code:: sh
+
+      issue:
+      suggestion:
+      question:
+      comment:
+
+Courtesy
+++++++++
+
+In general, it is important to be :ref:`courteous and respectful <pr-courtesy>`
+while also being clear and helpful to the author whose content you are
+reviewing. One way to do this is to be sure that you are always making comments
+about the *content* and never making comments about the *author*. You don't
+always have to follow this practice, but you should definitely use it when
+saying something that might otherwise be upsetting or contentious.
+
+.. example::
+
+   Bad: \*\*question\*\* Why did **you** use a semicolon to join two unrelated
+   clauses?
+
+   Good: \*\*suggestion: (blocking)\*\* Semicolons should only be used to
+   join two related independent clauses in place of a comma and coordinating
+   conjunction. Here's how you'd use a semicolon in this instance...
```

Comment:
This contraction is "you should" and therefore contains "you". I think this should be "make your best attempt" rather than "always".